src/view/screens/Search/Search.tsx

```diff
@@ -137,11 +137,11 @@ function SearchScreenSuggestedFollows() {
             ),
         )
       ).flat()
-
-      setSuggestions(
-        // dedupe
-        friendsOfFriends.filter(f => !friends.find(f2 => f.did === f2.did)),
+      const deduped = friendsOfFriends.filter(
+        (f, i) => friendsOfFriends.findIndex(f2 => f.did === f2.did) === i,
```

Comment:
Can we do `Array.from(new Set(friendsOfFriends))` to avoid loops inside a loop?

Reply:
Not with objects, but we could probably do this faster by creating a `Set(did, obj)` and reading values.

Reply:
yeah